falcon/testing/client.py

```diff
@@ -18,16 +18,17 @@
 WSGI callable, without having to stand up a WSGI server.
 """
 
-import json  # Don't use util.json so we can sanity-test compatibility with ujson output
 import platform
 import re
 import wsgiref.validate
 
 from six.moves import http_cookies
 
+from falcon.constants import MEDIA_JSON
 from falcon.testing import helpers
 from falcon.testing.srmock import StartResponseMock
 from falcon.util import CaseInsensitiveDict, http_date_to_dt, to_query_str
+from falcon.util import json as util_json
```

Comment:
Have gone this way since the pattern is sort of consistent with other parts of the project (cf ``falcon/response.py:37``: ``from falcon.util.uri import encode as uri_encode``, 
``falcon/response.py:38``: ``from falcon.util.uri import encode_value as uri_encode_value``).

But of course shout if this has to be changed :smiley: